### ISSUE TITLE
Display improvement for dec/hex/binary

### ISSUE THREAD

**Issue by Lecrapouille** [Qalculate/qalculate-gtk#58]
Like in gnome-calculator (see this [picture](https://en.wikipedia.org/wiki/GNOME_Calculator#/media/File:Screenshot_of_gnome-calculator_3.26.0_running_on_Arch_linux.png)) could be nice:
- to display (in smaller font) all dec/hex/octal/binary conversion directly in the result window
- display as well as the bit position counting
- adapt the font size of the result to fit the window (or simply split it into two lines)
- copy the result when clicking on it (or alternative clicking on bits of the binary displayed get the bit toggled).

Indeed with the current display `2^64  - 1` converting in:
- hexa: will display `0xFFFFFFFFFFFFFFFF` but could be better to more readable to display as `0xFFFF FFFF FFFF FFFF`  (with spaces)
- binary: will display a huge number which cannot fit in the window. Split it into two lines and add bit counter can help reading
```
   0000 0000 0000
     20   16   12
   111 1111 1111
     8    4    0
```

**Comment by antoniomolram** [comment]
I agree with this, I was searching for this too

**Comment by hanna-kn** [comment]
Part of this was implemented with the programming keypad in version 3.2 (see https://qalculate.github.io/manual/figures/programming-keypad.png).

**Comment by antoniomolram** [comment]
oh! thank you for your answer! I didn't know that. I use the version 0.9.9 that is the one available on Linux mint from the packages. I will try to update. Thank you again, and nice work!
